app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/building.py

```diff
@@ -0,0 +1,75 @@
+class Building:
```

Comment:
Forgot this in my PR. We'll need an `from ...python.id import id` at the top of this file